store/list.go

```diff
@@ -9,7 +9,8 @@ import (
 )
 
 var (
-	ErrIdNotExists = errors.New("id does not exist")
+	ErrIdNotExists  = errors.New("id does not exist")
+	ErrCpfNotExists = errors.New("cpf don't exists")
```

Comment:
seguindo o padrão da msg da linha acima seria
```suggestion
	ErrCpfNotExists = errors.New("cpf does not exist")
```